openapi-specs/mssp/adjust-specfile.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+for file in *.json; do
+
+    # /api/cloud/cspm/
+
+    sed -i '.bak' "s/\/api\/cloud\/cspm/\/prisma-cloud\/api\/mssp/g" $file
+    # /api/cloud -> /prisma-cloud/api/mssp
+    sed -i '.bak' "s/\/api\/cloud/\/prisma-cloud\/api\/mssp/g" $file
+
+
+    # rewrite the GLOBAL tag description when it is coming from the monolith file
+    tmp=$(mktemp)
+     jq '.info.description as $tag_desc| if($tag_desc!=null) then .tags[]?.description |= $tag_desc else . end' $file | \
+
+
+    # remove S2S headers or parameters if any
+    jq '.paths |= del(.[][].parameters[]? | select(.description!=null ) | select (.description | contains("S2S")))' | \
+
+    # delete code snippets
+    jq '.paths |= del(.[][]."x-codeSamples")' | \
+
+    # delete all 5xx error from endpoints
+    jq '.paths |= del(.[][].responses | .[keys[] | select(contains("5xx","5XX","500", "501", "502","503", "504", "505", "506", "507", "508","509","510","511", "529", "530","598","599"))])' | \
+
+    # add server urls
+    jq '.servers |= [
+        {"url":"https://api.prismacloud.io"},
+        {"url":"https://api2.prismacloud.io"},
+        {"url":"https://api3.prismacloud.io"},
+        {"url":"https://api4.prismacloud.io"},
+        {"url":"https://api.anz.prismacloud.io"},
+        {"url":"https://api.eu.prismacloud.io"},
+        {"url":"https://api2.eu.prismacloud.io"},
+        {"url":"https://api.gov.prismacloud.io"},
+        {"url":"https://api.prismacloud.cn"},
+        {"url":"https://api.ca.prismacloud.io"},
+        {"url":"https://api.sg.prismacloud.io"},
+        {"url":"https://api.uk.prismacloud.io"},
+        {"url":"https://api.ind.prismacloud.io"},
+        {"url":"https://api.jp.prismacloud.io"},
+        {"url":"https://api.fr.prismacloud.io"}]' | \
```

Comment:
hostname path should be `mssp-api.prismacloud.io` instead of `api.prismacloud.io.`